### PR TITLE
updates to compile with latest libraries

### PR DESCRIPTION
**Pull request by tacman**
This PR allows the application to work with PHP8 and the latest version of DoctrineBundle

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -63,13 +64,13 @@
     "autoload": {
         "psr-4": {
             "App\\": "src/",
-            "Acme\\FooBundle\\": "lib/acme/foo-bundle/"
+            "Survos\\LocationBundle\\": "lib/survos/location-bundle/"
```
Comment (msalsas) [inline composer.json:67]: Is this on purpose? I don't see the point of changing the namespace

File: composer.json [hunk 2]
```diff
@@ -63,13 +64,13 @@
...
         }
     },
     "autoload-dev": {
         "psr-4": {
             "App\\Tests\\": "tests/",
-            "Acme\\FooBundle\\": "lib/acme/foo-bundle/"
+            "Survos\\LocationBundle\\": "lib/survos/location-bundle/"
```
Comment (msalsas) [inline composer.json:73]: Same here

File: config/bundles.php [hunk 3]
```diff
@@ -15,5 +15,5 @@
     Symfony\WebpackEncoreBundle\WebpackEncoreBundle::class => ['all' => true],
     HtmlSanitizer\Bundle\HtmlSanitizerBundle::class => ['all' => true],
     Symfony\Bundle\MakerBundle\MakerBundle::class => ['dev' => true],
-    Acme\FooBundle\AcmeFooBundle::class => ['all' => true],
+    Survos\LocationBundle\SurvosLocationBundle::class => ['all' => true],
```
Comment (msalsas) [inline config/bundles.php:18]: This shouldn't be changed either

**Review by msalsas — CHANGES_REQUESTED** [review]
I don't see the point of changing the namespace to `Survos\LocationBundle`

**Comment by msalsas** [comment]
Could you also update the php version requirement in the README file? Thank you

**Comment by tacman** [comment]
Oops!  Yeah, the bundle I created got inserted unintentionally.  I'll tweak
the PR.

**Comment by msalsas** [comment]
The file `config/packages/acme_foo.yaml` should not be removed

**Comment by msalsas** [comment]
Thank you @tacman

**Pull request merged**